dietpi/dietpi-banner

```diff
@@ -56,6 +56,11 @@
 	#/////////////////////////////////////////////////////////////////////////////////////
 	# Main Loop
 	#/////////////////////////////////////////////////////////////////////////////////////
+	export TERM=${TERM:-dumb}
+	if (( "$TERM" == "dumb" )); then
```

Comment:
Thanks tdaitx,

I'll add this to the master branch manually. I'am unable to change/modify the code in this request as i dont have access to "patch-2".